packages/react-learn-card/package.json

```diff
@@ -70,14 +71,13 @@
   "dependencies": {
     "@learncard/core": "workspace:*",
     "date-fns": "^2.28.0",
+    "fitty": "^2.3.6",
     "numeral": "^2.0.6",
+    "pretty-bytes": "^6.0.0",
     "qrcode.react": "^3.0.2",
-    "react": "^18.2.0",
+    "react": "link:~/WeLibraryOS/wl-react/node_modules/react",
```

Comment:
We should probably change these local links back! (same for react-dom)

Reply:
Whoops, forgot about this 😅